tests/tck/features/match/PathExpr.feature

```diff
@@ -447,3 +447,34 @@ Feature: Basic match
       """
     Then the result should be, in any order:
       | v |
+
+  Scenario: different edge type in two pattern when apply join
+    When executing query:
+      """
+      MATCH (v:player{name: 'Tim Duncan'})-[e:like*2]->(n)
+        RETURN ()-[e:like*2]->(n)
+      """
+    Then the result should be, in any order:
+      | ()-[e:like*2]->(n) = ()-[e:like*2]->(n)                                                                                                                                                                                                                                                                                                   |
```

Comment:
why the column name is different with the query?

Reply:
fixed